ddtrace/contrib/cherrypy/__init__.py

```diff
@@ -0,0 +1,65 @@
+"""
+The Cherrypy trace middleware will track request timings.
+It uses the cherrypy hooks and creates a tool to track requests and errors
+
+
+Usage
+~~~~~
+To install the middleware, add::
+
+    from ddtrace import tracer
+    from ddtrace.contrib.cherrypy import TraceMiddleware
+
+and create a `TraceMiddleware` object::
+
+    traced_app = TraceMiddleware(cherrypy, tracer, service="my-cherrypy-app")
+
+
+Configuration
+~~~~~~~~~~~~~
+
+.. py:data:: ddtrace.config.cherrypy['distributed_tracing']
+
+   Whether to parse distributed tracing headers from requests received by your CherryPy app.
+
+   Can also be enabled with the ``DD_CHERRYPY_DISTRIBUTED_TRACING`` environment variable.
+
+   Default: ``True``
+
+.. py:data:: ddtrace.config.cherrypy['service_name']
```

Comment:
```suggestion
.. py:data:: ddtrace.config.cherrypy['service']
```

(both will actually work, but we are trying to move to just using `service` 😛)

Reply:
Now made it `service` across the board. Updated with master and pushed.